src/extensions/dnatco/confal-pyramids/property.ts

```diff
@@ -0,0 +1,167 @@
+/**
+ * Copyright (c) 2018-2020 mol* contributors, licensed under MIT, See LICENSE file for more info.
+ *
+ * @author Michal Malý <michal.maly@ibt.cas.cz>
+ * @author Jiří Černý <jiri.cerny@ibt.cas.cz>
+ */
+
+import { ModelsMap, Pyramid, PyramidsMapping } from './types';
+import { Column, Table } from '../../../mol-data/db';
+import { toTable } from '../../../mol-io/reader/cif/schema';
+import { CustomPropertyDescriptor } from '../../../mol-model/custom-property';
+import { Model, StructureElement } from '../../../mol-model/structure';
+import { CustomProperty } from '../../../mol-model-props/common/custom-property';
+import { CustomModelProperty } from '../../../mol-model-props/common/custom-model-property';
+import { PropertyWrapper } from '../../../mol-model-props/common/wrapper';
+import { ParamDefinition as PD } from '../../../mol-util/param-definition';
+import { MmcifFormat } from '../../../mol-model-formats/structure/mmcif';
+
+type ConfalPyramids = PropertyWrapper<{
+    map: ModelsMap,
+} | undefined >;
+
+export namespace ConfalPyramids {
+    export const Schema = {
+        ndb_struct_ntc_step: {
```

Comment:
Inclusion of these categories into CIF is currently being discussed. I am not sure if there is a public dictionary for this available just yet. While I was working on this code it occurred to me that the `ntc_step` and `ntc_step_summary` categories would be better merged into one.

Would you be okay with accepting the code as it is and updating the schema later once the final form makes it into CIF?

Reply:
This is totally fine as it is, especially since there are just a few fields. But it is a nice opportunity to leverage the automatic schema/code generation once the dictionary is published.